pages/implementation/destinations/segment-javascript.mdx

```diff
@@ -0,0 +1,138 @@
+---
+layout: 'guide'
+title: 'Use Segment in JavaScript'
+---
+
+Custom destination to send data from JavaScript to Segment
```

Comment:
Here we should also highlight that we do offer managed Segment SDK integration, same applies to the other Segment <> language docs